bundles/org.eclipse.core.resources/src/org/eclipse/core/resources/ResourcesPlugin.java

```diff
@@ -355,6 +356,14 @@ public final class ResourcesPlugin extends Plugin {
 	 */
 	public static final String PREF_MISSING_NATURE_MARKER_SEVERITY = "missingNatureMarkerSeverity"; //$NON-NLS-1$
 
+	/**
+	 * Name of a preference for configuring the marker severity in case project does
+	 * not specify its encoding
+	 *
+	 * @since 3.18
+	 */
+	public static final String PREF_MISSING_ENCODING_MARKER_SEVERITY = "missingEncodingMarkerSeverity"; //$NON-NLS-1$
```

Comment:
Could you please add a note about expected / allowed values (especially "ignored" one). It is not clear that other values than `IMarker.SEVERITY_` are also accepted.

Reply:
Added the info.

Thought: I would be nice to have a severity constant for this like {{IMarker.SEVERITY_IGNORE}}. But I understand that that would require much more discussion as that would affect API.
I "copied" that approach with ``severity < 0`` (again, as suggested when I started this issue) from ``CheckMissingNaturesListener`` (see ``org.eclipse.core.internal.resources.CheckMissingNaturesListener.updateMarkers(Collection<IProject>)``)